app/usecases/attachments/unlink.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Usecases
+  module Attachments
+    class Unlink
+      attr_reader :attachment
+
+      def initialize(attachment)
+        @attachment = attachment
+      end
+
+      def self.execute!(attachment)
+        new(attachment).execute!
+      end
+
+      def execute!
+        attachment.tap do |a|
+          a.attachable_id = nil
+          a.attachable_type = 'Container'
```

Comment:
why?

Reply:
why what? can you be more specific please?

Reply:
why the attachable_type set to 'Container' when unlink?

Reply:
That is what we wondered as well... please see https://github.com/ComPlat/chemotion_ELN/commit/e04c83b047af65f528e965023341ba04b3a070cd